src/components/Dropdown/Dropdown.vue

```diff
@@ -22,6 +22,10 @@
                 <slot name="menuPrepend" />
             </template>
 
+            <template #items>
+                <slot name="menuItems" />
+            </template>
```

Comment:
I think I would not be able to do the following:
```
<Dropdown>
    <template #item="{item}">
        {{ item.label }}
```
I think this also applies to `FormDropdown`. Can we have a nice eligant and flexible way to pass these slots to children?